src/count/gulpfile.js

```diff
@@ -0,0 +1,48 @@
+const standaloneName = 'oCommentCount';
```

Comment:
This file can probably be removed as I think it will be handled by the o-comments version